src/postgres/src/test/regress/yb_pg_misc_serial_schedule

```diff
@@ -18,14 +17,13 @@ test: yb_pg_create_table
 
 test: yb_pg_create_function_2
 
-# Dependency for `yb_pg_select`
+# Dependency for `yb_pg_select`, 'yb_pg_groupingsets', 'yb_pg_random', 'yb_pg_limit'
 test: yb_pg_copy
 
 test: yb_pg_create_operator
 
-# Dependency for `yb_pg_aggregates`
-test: yb_pg_create_aggregate
-
 test: yb_pg_errors
 test: yb_pg_select
```

Comment:
```suggestion
test: yb_pg_select
test: yb_pg_errors
```

This is not part of your change but worth correcting.  Order using `serial_schedule`.